vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/opensds/Constants.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package org.opensds.vmware.ngc.adapters.opensds;
+
+import java.util.Properties;
+
+ enum UNIT_TYPE {
+
+    /**
+     * byte
+     */
+    Byte(1L),
+
+    /**
+     * KB
+     */
+    KB(1024L),
+
+    /**
+     * MB
+     */
+    MB(1024L * 1024L),
+
+    /**
```

Comment:
> Can these constants be defined with actual value after multiplication;
> 1024_1024 = 1048576 1024_1024*1024 = 1073741824
> otherwwise, everytime we call this constant.. it does multiplication

@satya-gorli , this is for more readability, so that code reader will undestand this is  GB (1024*1024*1024) but not (1000*1000*1000).  muliplication cost is neglactable as JVM optimizes enums.

Reply:
ok fine